sickrage/providers/nzb/NZBProvider.py

```diff
@@ -39,23 +39,13 @@ def _get_result(self, episodes):
         return NZBSearchResult(episodes)
 
     def _get_size(self, item):
-        try:
-            size = item.get('links')[1].get('length', -1)
-        except (AttributeError, IndexError, TypeError):
-            size = -1
```

Comment:
Whel actually those exceptions can raise.  None[1]. Buut probly just the indexError.